### DISCUSSION TITLE
Elementor Form : Checkbox "required" field option

### DISCUSSION THREAD

**Discussion by thanhkatz** [elementor/elementor#21227] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I was unable to answer my client's demand : Make the Elementor form checkboxes widget "required" 

### Describe the solution you'd like

It would be nice to have a "required" option as the other widgets. I could be fantastic to enable it for all widgets. 

### Describe alternatives you've considered

I considered using the others choices widget, radio and select. But it wasn't suitable in terms of design.

**Comment by ricohlmann** [comment]
+1 
i don't know why the other tickets regarding this issue are unsolved and closed :(

**Comment by qurgh** [comment]
They will happily work on a new version that no one has asked for, yet they can't even add a required tag to a checkbox that the users have been asking for years and years...
